openupgrade_scripts/scripts/website/14.0.1.0/post-migration.py

```diff
@@ -44,3 +80,10 @@ def migrate(env, version):
     openupgrade.delete_records_safely_by_xml_id(env, ["website.aboutus_page"])
     website_cookie_notice_post_migration(env)
     convert_html_string(env)
+    """ This call is commented out as this script causes conflicts with inheritance
+    in migrations to later versions. In v14 a duplicate of website.layout with
+    associated website is made when the company name is modified in the footer,
+    this script replicates that behaviour in the migration but in later versions
+    it can be a problem of inheritance. However, it can be uncommented and used if
+    deemed appropriate. """
```

Comment:
This can't be a text, but several commented lines:

```suggestion
    # This call is commented out as this script causes conflicts with inheritance
    # in migrations to later versions. In v14 a duplicate of website.layout with
    # associated website is made when the company name is modified in the footer,
    # this script replicates that behaviour in the migration but in later versions
    # it can be a problem of inheritance. However, it can be uncommented and used if
    # deemed appropriate. """
```